# Motor speed in QPPS
#int32 m1_speed
#int32 m2_speed
int32 m1_qpps
int32 m2_qpps

# Motor distance in encoder tick count
#uint32 m1_dist
#uint32 m2_dist

# Motor acceleration in QPPS per second
#uint32 accel

# Max seconds before automatically stopping
uint32 max_secs
